Maman11/Q1/my_add.c

```diff
@@ -0,0 +1,55 @@
+#include <stdio.h>
+#include <stdlib.h>
+#define BYTE_SIZE 8
+
+void print_binary(int num) {
+    if (num > 1) {
+        print_binary(num / 2);  // Recursive call for higher bits
```

Comment:
This type of comments will give you warnings in c90 look on the make file that I did (you should use the flags -Wall -ansi -pedantic)

c90 (old c lang) uses `/* comment */` to comment so just check for warnings